source/MaterialXTest/MaterialXFormat/VersionFormat.cpp

```diff
@@ -0,0 +1,108 @@
+#include <MaterialXTest/Catch/catch.hpp>
```

Comment:
Can you move this to MaterialXRuntime as if you don't build MaterialXRuntime (a build option) then this test won't build. 
Thanks.